packages/transaction-controller/src/types.ts

```diff
@@ -689,3 +689,19 @@ export type DefaultGasEstimates = {
    */
   maxPriorityFeePerGas?: string;
 };
+
+/**
+ * Describes the event types emitted by the transaction controller.
+ */
+export enum TransactionEvent {
```

Comment:
Since we have the extended `EventEmitter` interface to enforce the event names and associated arguments, do we also need this enum?

Any instance where we want to use an event name should be covered by the interface meaning is there any value in putting the strings themselves in an enum?

Auto-complete will recommend the event names also based on the interface.

Reply:
Sure, these are removed.